### DISCUSSION TITLE
Leaderboard text color can be hard to read at 100% background dim

### DISCUSSION THREAD

**Discussion by Flamiii** [ppy/osu#20537] [Q&A]
![image](https://user-images.githubusercontent.com/74088315/193160369-34f9a546-d207-4ba9-8289-909aceef227b.png)
![image](https://user-images.githubusercontent.com/74088315/193160349-d67a3dd1-96f1-4cea-bdca-fbdbc5f0814b.png)

Personally, I feel that the text color that is used for the player highlighted in yellow is a bit too dark to be clearly legible at 100% background dim. At 100% background dim, the surrounding color is a fairly dark shade of yellow, and the text itself seems to be a pretty dark shade of blue, and to me (I'm not colorblind), I find this color combination to be pretty hard to read when compared to the bright white text surrounded by dark shades of green and blue above.

**Comment by peppy** [comment]
The leaderboard design is likely not final.

For not you could decrease your dim to resolve the issue.

**Reply by peppy** [reply]
@arflyte

**Comment by iwa** [comment]
Another input about the leaderboard, wouldn't it be better to switch the green & gold color?
I think it would make more sense for everyone that way, as gold is often refered as for the #1
Green would make a good "active" color instead
